aws-lc-sys/builder/cc_builder.rs

```diff
@@ -188,7 +188,7 @@ impl CcBuilder {
             env::set_var("CFLAGS", cflags);
         }
 
-        if target_os() == "macos" && target_arch() == "x86_64" {
+        if target_os() == "macos" {
             // This compiler error has only been seen on MacOS x86_64:
```

Comment:
Does this comment need updating

Reply:
It's still a true statement, but I think the reason for it is that for MacOS 13 we only use x86-64 runners and not ARM64.